reckless_lib/src/plugin.rs

```diff
@@ -86,7 +88,19 @@ impl Plugin {
             if let Some(script) = &conf.plugin.install {
                 let cmds = script.split("\n"); // Check if the script contains `\`
                 for cmd in cmds {
-                    // FIXME: run command
+                    let cmd_result = Command::new(cmd)
+                        .current_dir(self.path.clone())
+                        .output()
+                        .await;
+                    match cmd_result {
+                        Ok(_) => return Ok(self.path.clone()),
```

Comment:
```suggestion
                        Ok(_) => {},
```

We need to return the path of the main file and not the root of the plugin, otherwise, we do not solve our problem.

N.B it is already done at line  105